hbase-common/src/main/java/org/apache/hadoop/hbase/TableName.java

```diff
@@ -242,6 +255,9 @@ public static void isLegalNamespaceName(final byte[] namespaceName,
       // A znode named "zookeeper" is disallowed by zookeeper.
       throw new IllegalArgumentException("Tables may not be named '" + DISALLOWED_TABLE_NAME + "'");
     }
+    if(namespaceName.length > MAX_NAMESPACE_LENGTH){
```

Comment:
See background code. It has a space after the 'if' and before the '('.  Ditto on end.

Can you add a test for this nice addition? Thanks.

Reply:
Any update here @tomZou12 ? Thanks.